airflow/pod_templates/medium_task_template.yaml

```
@@ -93,3 +93,12 @@ spec:
     - name: config
       configMap:
         name: airflow-config
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: polars-unsafe # OKD worker0 (should not impact Openshift)
```

```suggestion
              - key: polars-unsafe # OKD worker0 (should not impact BC Government Openshift)
```
Maybe just to make it even more clear

---

Good shout - I will do that.